src/agora/radio/radio_lib.h

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file radio_lib.h
+ * @brief Declaration file for the RadioConfig class.
+ */
+#ifndef RADIO_LIB_H_
+#define RADIO_LIB_H_
+
+#include <complex>
+#include <cstdlib>
+#include <memory>
+#include <vector>
+
+#include "SoapySDR/Device.hpp"
+#include "armadillo"
+#include "config.h"
+#include "memory_manage.h"
+#include "radio.h"
+
+class RadioConfig {
```

Comment:
name the file as class name?

Reply:
Didn't change the file name or the class name.  We can update both (probably) in future prs